Rightpoint/ios-template
# Switch code signing to use fastlane match

## Notes:
This PR updates the fast file to sign with match. It shifts export config from the fast file toward the xcconfig and build settings to clean up the fast file a bit. Additionally, it uses the RP enterprise team id and wild card UUID so RP develop and sprint builds are able to code sign without additional config out of the box.

### Changes:
- remove old signing dir
- adds match file and configures fastfile to sign with match
- removes old fastlane plugin code
- re structures xcconfigs to support new signing paradigm, defaults to
RP enterprise credentials
- regenerated the template with these changes
- `fastlane develop` & `fastlane sprint` build and sign out of the box!

## Testing instructions:
- run cookie cutter as specified in [testing your changes](https://github.com/Rightpoint/ios-template#testing-your-changes).
- ensure your git profile has access to the RP match repo.
- `bundle exec fastlane develop` and `bundle exec fastlane sprint` should build and sign the application succssfully (you can terminate the script when it prompts you for a hockey key).
